Bottom: bfe38f63aa11c17a65b7d2fde6a8a204be8b3ab0
Top:    444294b160bc9e69ff9b439d25e7fa3bd8abfd8d
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-13 10:53:14 -0500

Extract email accounts to group vars

Email accounts belonging to eother services were present in exim4's defaults.


---
diff --git a/ansible-playbook/group_vars/all/email.yml b/ansible-playbook/group_vars/all/email.yml
new file mode 100644
index 00000000..fe6c710a
--- /dev/null
+++ b/ansible-playbook/group_vars/all/email.yml
@@ -0,0 +1,10 @@
+email_accounts:
+  wiki:
+    user: wiki
+    password: '{{ wiki_email_password }}'
+  gitlab:
+    user: gitlab
+    password: '{{ gitlab_email_password }}'
+  mantisbt:
+    user: mantis
+    password: '{{ mantisbt_email_password }}'
diff --git a/roles/docker-exim4/defaults/main.yml b/roles/docker-exim4/defaults/main.yml
index dd4c2eca..2a0e9f7e 100644
--- a/roles/docker-exim4/defaults/main.yml
+++ b/roles/docker-exim4/defaults/main.yml
@@ -35,15 +35,6 @@ exim4_syslog_datavolume_dir: '/var/log/docker_container/{{ exim4_container_name
 
 
 email_accounts:
-  wiki:
-    user: wiki
-    password: '{{ wiki_email_password }}'
-  gitlab:
-    user: gitlab
-    password: '{{ gitlab_email_password }}'
-  mantisbt:
-    user: mantis
-    password: '{{ mantisbt_email_password }}'
 
 # Network names
 docker_network_frontend: 'frontend'
